src/ReflectionParameter.php

```diff
@@ -44,12 +54,35 @@ class ReflectionParameter implements \Reflector
     private $types;
 
     /**
+     * @var Type
+     */
+    private $typeHint;
```

Comment:
Don't need a default value, we can only instantiate `ReflectionParameter::createFromNode` which always sets `$typeHint` value